src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -40,7 +40,7 @@ public static IWebHost BuildWebHost(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
         {
             return AspNetCore.WebHost.CreateDefaultBuilder(args)
-                .UseKestrel(o =>
```

Comment:
Is this change required? UseKestrel should still work, correct?

Reply:
UseKestrel does not work, ASP.NET Core folks said to change it to ConfigureKestrel for 2.2

Reply:
Sounds good. It would be good to have more context on this as it's not clear why that `UseKestrel` would no longer work after moving to 2.2

Reply:
`UseKestrel` overrides `IServer` implementation every time it's called.

So what happens is `AspNetCore.WebHost.CreateDefaultBuilder` calls `UseIIS()` to set server to InProcess ANCM and calling `UseKestrel()` resets that. `ConfigureKestrel` just changes options without overriding servers.

Reply:
Thanks for the clarification, @pakrym. That makes sense.